rtos/Queue.h

```diff
@@ -301,12 +304,34 @@ class Queue : private mbed::NonCopyable<Queue<T, queue_sz> > {
      *
      * @note  You may call this function from ISR context if the millisec
      *        parameter is set to 0.
-     * @deprecated Pass a chrono duration, not an integer millisecond count. For example use `5s` rather than `5000`.
+     * @deprecated Replaced with try_get and try_get_for. In future get will be an untimed blocking call.
      */
-    MBED_DEPRECATED_SINCE("mbed-os-6.0.0", "Pass a chrono duration, not an integer millisecond count. For example use `5s` rather than `5000`.")
-    osEvent get(uint32_t millisec)
+    MBED_DEPRECATED_SINCE("mbed-os-6.0.0", "Replaced with try_get and try_get_for. In future get will be an untimed blocking call.")
```

Comment:
In the future the function signature should also change to take a ` T **` parameter

Reply:
The untimed `get` can actually just be `T *get()` - it can return it directly.